components/memberstack/actions/update-member/update-member.mjs

```diff
@@ -0,0 +1,50 @@
+import memberstack from "../../memberstack.app.mjs";
+import pickBy from "lodash.pickby";
+
+export default {
+  key: "memberstack-update-member",
+  name: "Update Member",
+  description: "Updates a member. [See the docs](https://memberstack.notion.site/Admin-API-5b9233507d734091bd6ed604fb893bb8)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    memberstack,
+    memberId: {
+      propDefinition: [
+        memberstack,
+        "memberId",
+      ],
+    },
+    customFields: {
+      propDefinition: [
+        memberstack,
+        "customFields",
+      ],
+    },
+    metaData: {
+      propDefinition: [
+        memberstack,
+        "metaData",
+      ],
+    },
+  },
+  async run({ $ }) {
+    let response;
+    try {
+      const { data } = await this.memberstack.updateMember({
+        id: this.memberId,
+        data: pickBy({
+          customFields: this.customFields,
+          metaData: this.metaData,
+        }),
+      });
+      response = data;
+    } catch (e) {
+      throw new Error(e.message);
```

Comment:
Do we need this `try/catch` that just throws a new error? It wouldn't be better simply to let it throw automatically?